common/util.go

```diff
@@ -79,17 +79,13 @@ const (
 	contextExpireThreshold = 10 * time.Millisecond
 
 	// FailureReasonCompleteResultExceedsLimit is failureReason for complete result exceeds limit
-	FailureReasonCompleteResultExceedsLimit = "COMPLETE_RESULT_EXCEEDS_LIMIT"
+	FailureReasonCompleteResultExceedsLimit = "Complete result exceeds size limit."
 	// FailureReasonFailureDetailsExceedsLimit is failureReason for failure details exceeds limit
-	FailureReasonFailureDetailsExceedsLimit = "FAILURE_DETAILS_EXCEEDS_LIMIT"
+	FailureReasonFailureExceedsLimit = "Failure exceeds size limit."
 	// FailureReasonCancelDetailsExceedsLimit is failureReason for cancel details exceeds limit
-	FailureReasonCancelDetailsExceedsLimit = "CANCEL_DETAILS_EXCEEDS_LIMIT"
+	FailureReasonCancelDetailsExceedsLimit = "Cancel details exceed size limit."
 	// FailureReasonHeartbeatExceedsLimit is failureReason for heartbeat exceeds limit
-	FailureReasonHeartbeatExceedsLimit = "HEARTBEAT_EXCEEDS_LIMIT"
-	// FailureReasonDecisionBlobSizeExceedsLimit is the failureReason for decision blob exceeds size limit
-	FailureReasonDecisionBlobSizeExceedsLimit = "DECISION_BLOB_SIZE_EXCEEDS_LIMIT"
-	// FailureReasonSizeExceedsLimit is reason to fail workflow when history size or count exceed limit
-	FailureReasonSizeExceedsLimit = "HISTORY_EXCEEDS_LIMIT"
```

Comment:
Why is this gone? This is a legitimate failure case.

Reply:
Restored `FailureReasonSizeExceedsLimit`. And `FailureReasonDecisionBlobSizeExceedsLimit` is not needed anymore because there are 8 different messages used for this type of failure. And constant was used to determinate retryable, which is seprate field now.